Velaptor/Services/IDotnetService.cs

```diff
@@ -0,0 +1,27 @@
+namespace Velaptor.Services;
+
+/// <summary>
+/// Invokes Dotnet functions.
+/// </summary>
+public interface IDotnetService
+{
+    /// <summary>
+    /// Invokes System.GC.KeepAlive method.
+    /// </summary>
+    /// <param name="obj">The object to reference.</param>
+    public void GCKeepAlive(object? obj);
+
+    /// <summary>
+    /// Invokes System.Runtime.InteropServices.Marshal.StringToHGlobalAnsi method.
+    /// </summary>
+    /// <param name="s">A managed string to be copied.</param>
+    /// <returns>The address, in unmanaged memory, to where s was copied, or 0 if s is null.</returns>
```

Comment:
Please refactor the code doc to reference the original functionality that it wraps.

```suggestion
    /// <inheritdoc cref="Marshal.StringToHGlobalAnsi"/>
```